docs/tutorials/azure-private-dns.md

```diff
@@ -216,7 +216,7 @@ subjects:
   name: externaldns
   namespace: default
 ---
-apiVersion: extensions/v1beta1
+apiVersion: apps/v1
```

Comment:
Afaik, switching to "apps/v1" also requires to define "matchLabels" under spec. Would you mind checking that as well? Otherwise the docs might contain invalid manifests which would confuse people.

Reply:
Thanks for the review. I've added selectors to all the deployments I updated.